amarachukwu-agbo/SMS-management-API
# #165412347 Set up node and express

#### What does this PR do?
Sets up `Node` and `Express`

#### Description of Task to be completed?
- Install dependencies
- Create entry file
- Set up `babel` and `codeclimate` config file
- Create `gitignore` file

#### How should this be manually tested?
- Clone the repo and install application dependencies.
 - Run command  `yarn start:dev`. 
- Open Postman and make a request `GET http://localhost:4000/`
- You should get a 200 response

#### Any background context you want to provide?
N/A

#### What are the relevant pivotal tracker stories?
[#165412347](https://www.pivotaltracker.com/story/show/165412347)

#### Screenshots (if appropriate)
N/A

#### Questions: 
N/A
